docs/requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --allow-unsafe --strip-extras docs/requirements.in
```

Comment:
It's best to use the same working dir as dependabot to avoid this header being updated back and forth all the time...

Reply:
Though, in the last bot PR (#2043), things worked as expected?

Reply:
Ah, interesting. Anyways, please don't mix infra changes with the functional ones. We don't have an ability/infra to make two separate changelog entries per PR. With atomic PRs, this wouldn't be a problem.

Reply:
Marking this as resolved to unblock.